container/container_windows.go

```diff
@@ -110,3 +110,8 @@ func (container *Container) BuildHostnameFile() error {
 func (container *Container) canMountFS() bool {
 	return !containertypes.Isolation.IsHyperV(container.HostConfig.Isolation)
 }
+
+// EnableServiceDiscoveryOnDefaultNetwork Enable service discovery on default network
+func (container *Container) EnableServiceDiscoveryOnDefaultNetwork() bool {
+	return true
```

Comment:
I was originally thinking of not supporting this. But the only reason why we dont support this in Linux today is for backward compatibility reasons. With this being the first iteration of native docker platform on windows, I think it is appropriate to allow it. 👍